application/event_processor/event_processor.py

```diff
@@ -164,8 +167,17 @@ def lambda_handler(event: SQSEvent, context: LambdaContext) -> None:
     message_received_pretty = "%s.%03d" % (strftime("%Y-%m-%d %H:%M:%S", gmtime(s)), ms)
     logger.append_keys(message_received=message_received_pretty)
     logger.info("Change Event received", extra={"event": event})
+    metrics.set_namespace("UEC-DOS-INT")
+    metrics.set_property("level", "INFO")
+    metrics.set_property("function_name", context.function_name)
+    metrics.set_property("message_received", message_received_pretty)
     db_latest_sequence_number = get_latest_sequence_id_for_a_given_odscode_from_dynamodb(change_event["ODSCode"])
     record_id = add_change_request_to_dynamodb(change_event, sequence_number, sqs_timestamp)
+
+    metrics.set_property("correlation_id", logger.get_correlation_id())
```

Comment:
Won't this create a seperate metric for each run of the event_processor? As the correlation_id is unique on each run

Reply:
no set_property just adds the field to the log in that metric scope, set_dimension would create a metric but set_property just adds context to the cloud watch logs it creates